scripts/variantstore/wdl/GvsPopulateAltAllele.wdl

```diff
@@ -166,10 +221,11 @@ task PopulateAltAlleleTable {
       --call_set_identifier ~{call_set_identifier} \
       --query_project ~{project_id} \
       --vet_table_name ~{vet_table_name} \
-      --fq_dataset ~{project_id}.~{dataset_name}
+      --fq_dataset ~{project_id}.~{dataset_name} \
+      --max_sample_id ~{max_sample_id} \
   >>>
   runtime {
-    docker: "us.gcr.io/broad-dsde-methods/variantstore:ah_var_store_2022_08_16"
+    docker: "us.gcr.io/broad-dsde-methods/variantstore:rsa_vs_52_incremental_alt_allele_2022_08_17"
```

Comment:
Once ready to merge, can your create and use 'ah_var_store_2022_08_22' (or whatever date). And update everywhere?